lib/_debugger.js

```diff
@@ -126,7 +126,8 @@ Protocol.prototype.execute = function(d) {
       break;
 
     default:
-      throw new Error('Unknown state');
+      const errors = require('internal/errors');
```

Comment:
Lazy loading is pretty common in core code. Though I'm not sure how much it actually saves us now.